templates/base.html

```diff
@@ -26,16 +26,14 @@
     {% include 'snippets/head_meta.html' %}
     <meta name="google-site-verification" content="pS2kpksxg6JeC90IvA8BHVsFuK_6b7J_vARVLqqu7ck" />
     {% block extra-head %}{% endblock %}
-
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
+        integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+        crossorigin="anonymous"></script>
+    {% include "snippets/editoria11y.html" %} {# include before local styles #}
```

Comment:
just occurred to me that this might work without even including jQuery, if it's already a dependency of semantic UI (and they might even conflict if it is). did you see what happens if you don't include jQuery?

Reply:
we already had the jquery include, I just moved where it was so it would be available before editoria11y — I had to move up the editoria11y include so our local css would override it, since the editoria11y styles are already set as important 😕  (also this is cdhweb, not ppa, so no semantic ui 🙂 )

Reply:
🤦🏻 ignore me